composablejs/packages/types/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "./dist"
+  },
+  "include": ["./src/**/*"],
+  "exclude": [
+    "node_modules",
+    "dist"
+  ]
+}
```

Comment:
Can we add a comment why we cannot put this on strict yet? Or is that no longer the case?

Reply:
The child package configuration are supposed to get their strict settings through the root tsconfig.
But thanks for bringing this up, I realized i can stricten & clean this setup a bit better.